web/src/compose_setup.js

```diff
@@ -583,6 +595,7 @@ export function initialize() {
         } else {
             compose_notifications.maybe_show_one_time_interleaved_view_messages_fading_banner();
         }
+        compose_recipient.maybe_show_go_to_conversation_button_intro_tooltip();
```

Comment:
Should this be called in `setTimeout` to avoid race with our code which modified the class of the go to conversation button?